test/package/README.md

```diff
@@ -0,0 +1,62 @@
+# Package Testing
+
+Packaging tests use Vagrant virtual machines as hosts and Ansible for provisioning and assertions. Kibana distributions are copied from the target folder into each VM and installed, along with required dependencies.
+
+## Setup
+
+- [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+
+  ```
+  # Ubuntu
+  sudo apt-get install python3-pip
+  pip3 install --user ansible
+
+  # Darwin
+  brew install python3
+  pip3 install --user ansible
+  ```
+
+- [ansible.posix.firewalld](https://docs.ansible.com/ansible/latest/collections/ansible/posix/firewalld_module.html)
+  ```
+  ansible-galaxy collection install ansible.posix
+  ```
+- [Vagrant](https://www.vagrantup.com/downloads)
+- [Virtualbox](https://www.virtualbox.org/wiki/Downloads)
+
+## Machines
+
+| Hostname | IP           | Description                           |
+| -------- | ------------ | ------------------------------------- |
+| deb      | 192.168.50.5 | Installation of Kibana's deb package  |
+| rpm      | 192.168.50.6 | Installation of Kibana's rpm package  |
+| docker   | 192.168.50.7 | Installation of Kibana's docker image |
+
+## Running
+
+```
+# Build distributions
+node scripts/build --all-platforms --debug --no-oss
+
+# Setup virtual machine and networking
+vagrant up <hostname> --no-provision
+
+# Install Kibana and run OS level tests
+# This step can be repeated when adding new tests, it ensures machine state - installations won't run twice
+vagrant provision <hostname>
```

Comment:
Is there any way to know when Kibana should be up and ready?

Reply:
There's a test that waits for port 5601 to be listening over TCP - I haven't had any issues with it so far, but we could add another to listen over http.